### PR TITLE
feat: add methods table

### LINKED ISSUE #103: docs: list of methods
*Should we include a list of methods* at the end of `execution_workflows/README.md`, that can be linked to in the presentation (and for documentation).    
Possibly useful columns:
- Method name
- link to publication
- link to repository
- type (identification, quantification, differential usage)
- status in APAeval (ready, in progress, to do, on hold, dismissed, etc.) (check PRs, issues, slack execution_workflows)
- present in OEB (Yes/No)
- installation (Bioconductor, bioconda, ...?)
- ~Notes (don't know, e.g. R based, using DESeq, only method that can take your dog for a walk)~
- can take your dog for a walk (Yes/No/depends)

### PR DESCRIPTION
**Pull request by chelseaherdman**
Fixes #103 

## Checklist

- [x] I have performed a self-review of my own code
- [x] My code follows the templates/style guidelines of the repository
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested my code

### CODE REVIEW

**Comment by chelseaherdman** [comment]
@uniqueg can you check this methods table in the readme please? I think i did what you and @ninsch3000 were asking with the columns

File: execution_workflows/README.md
Comment (uniqueg) [inline execution_workflows/README.md:96]: Status: I would say `On hold`

File: execution_workflows/README.md
Comment (uniqueg) [inline execution_workflows/README.md:82]: Also, I've thought a little bit about "Installation", and I think that's not what's relevant to most people as it would rarely have anyone change their mind about using a tool. I think what would be better is to have a column "Packaging" with one or more of the following options:
- `Conda: [channels]` (list `bioconda`, if available; if available but a version from another channel was downloaded more often, list that one, too)
- `Containers: [maintainers]` (list one or both of `BioContainers` and author's `official` repository)
- `Workflow: [workflow language]` (list one of `CWL`, `Galaxy`, `Nextflow`, `Snakemake`, `WDL` or `other`)
- `Other`, optionally with a package repository:
  - `PyPI`
  - `CRAN`
  - `Bioconductor`
  - `GitHub` - but only if it uses [GitHub _Packages_](https://github.com/features/packages) - it's a specific tab that only shows up if repos support it, which is not (yet) very commonly used - not just if the code is on GitHub any old way!
  - maybe there are others, but most likely not

Examples:

* `Conda: bioconda; Containers: BioContainers, official; Other: PyPI`
* `Workflow: Snakemake`
* `Workflow: other`
* `Other: Bioconductor`
* `Other`

Each channel, registry or repository should be linked.

As all of this is a lot of work, I think it's best to create an issue for that and first merge this version

File: execution_workflows/README.md
Comment (uniqueg) [reply execution_workflows/README.md:96]: Or rather (see above) just link to the issue

**Review by uniqueg — CHANGES_REQUESTED** [review]
Looks great, especially the last column :heart:

Still added quite a few comments (sorry I didn't think too hard about it earler). I think it's best to put some of these into one or more issues for the future, as they will be quite a lot of (dull) work... ;-)

File: execution_workflows/README.md
Comment (uniqueg) [inline execution_workflows/README.md:82]: Maybe you could replace the word `dog` with `:dog:`, `:dog2:` or `:poodle:` :dog: :dog2: :poodle: 😜

Comment (chelseaherdman) [reply execution_workflows/README.md:82]: updated with issue links

File: execution_workflows/README.md
Comment (chelseaherdman) [reply execution_workflows/README.md:84]: yup, done

Comment (chelseaherdman) [reply execution_workflows/README.md:84]: will put as new issue for now

Comment (chelseaherdman) [reply execution_workflows/README.md:84]: done

File: execution_workflows/README.md
Comment (chelseaherdman) [reply execution_workflows/README.md:82]: will put as new issue

Comment (chelseaherdman) [reply execution_workflows/README.md:82]: 😂  done

**Comment by chelseaherdman** [comment]
I will add two new issues for the remaining tasks (installation and publication links) but I think it can be merged now so we at least have something there.

**Comment by chelseaherdman** [comment]
I'll actually need help updating this properly tomorrow.

**Comment by uniqueg** [comment]
Sure, will help you :)

**Comment by chelseaherdman** [comment]
Closing this. New PR #133

**Pull request closed without merging**
